packages/valory/services/oracle_hardhat/service.yaml

```diff
@@ -14,10 +14,16 @@ extra:
   setup_args:
     args: &id001
       setup:
-        safe_contract_address: []
+        all_participants:
+        - - '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'
+          - '0x70997970C51812dc3A010C7d01b50e0d17dc79C8'
+          - '0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC'
+          - '0x90F79bf6EB2c4f870365E785982E1f101E93b906'
```

Comment:
When deploying, the `keys.json` needs to contain these addresses and their corresponding private keys. These are the ones in the registries image for service id 1.

Maybe we would need to update the keys.json suggestion in the documentation to make this easier for the users @jmoreira-valory.

Reply:
Please, update the keys and addresses in the index.md file in this PR. I will review the text in another PR.

Reply:
To save some CI time, could you edit them on the PR you mentioned? They should be:

```json
[
  {
      "address": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
      "private_key": "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
  },
  {
      "address": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
      "private_key": "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
  },
  {
      "address": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
      "private_key": "0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a"
  },
  {
      "address": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
      "private_key": "0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6"
  }
]
```